tests/data_context/store/test_store_backends.py

```diff
@@ -1598,19 +1598,20 @@ def test_GeCloudStoreBackend_casts_str_resource_type_to_GeCloudRESTResource() ->
 def test_InlineStoreBackend(empty_data_context: DataContext) -> None:
```

Comment:
These tests needed a lot of fixing up due to `resource_type` being a constructor arg instead of a key arg.